How-To-Data-Query-Templates.md

```diff
@@ -196,5 +196,10 @@ Cacti results in the following:
 + Inserting index data for field 'Index' [value='0']
 ```
 
+## A note on Textual SNMP MIBS and SPINE
+
+Please be aware the SPINE does not parse out textual Mibs if you were to put a SNMP string i.e interfaces.mib.1.1.1.1.1 this will fail
```

Comment:
This line should be no more than 80 characters in length and is what caused travis to fail.